packages/tideline/js/plot/message.js

```diff
@@ -120,19 +120,20 @@ module.exports = function(pool, opts) {
     });
 
     var foGroup = tooltip.foGroup;
+    const msgDate = format.datestamp(d.normalTime, d.displayOffset);
+    const msgTime = format.timestamp(d.normalTime, d.displayOffset);
+    const msgDateTime = `<span class="fromto">${t('{{date}} - {{time}}', {date: msgDate, time: msgTime})}</span>`;
```

Comment:
I don't see why we would need this entry translated as the dates are already formatted with translated formats
```suggestion
    const msgDateTime = `<span class="fromto">${msgDate} - ${msgTime}</span>`;
```